src/executor_v2/drop.rs

```diff
@@ -0,0 +1,23 @@
+// Copyright 2022 RisingLight Project Authors. Licensed under Apache-2.0.
+
+use std::sync::Arc;
+
+use super::*;
+use crate::binder_v2::{BoundDrop, Object};
+use crate::storage::Storage;
+
+/// The executor of `drop` statement.
+pub struct DropExecutor<S: Storage> {
+    pub plan: BoundDrop,
+    pub storage: Arc<S>,
+}
+
+impl<S: Storage> DropExecutor<S> {
```

Comment:
Indeed now I'm inclined not to have DropTableExecutor / CreateTableExecutor like what DuckDB did... create / drop table are pure catalog operations and can be handled after binding.